README.md

```diff
@@ -210,4 +210,20 @@ Hue is given in degrees (0-360) and saturation and value in percentage.
 print(bulb.hsv)
 if bulb.is_color:
    bulb.hsv = (180, 100, 100) # set to cyan
+```
+
+## Development Setup
+
+### Docker
+
+Install Docker for your system [Docker for Mac](https://docs.docker.com/docker-for-mac/), [Docker for Windows](https://docs.docker.com/docker-for-windows/), [Docker for Linux](https://docs.docker.com/engine/installation/#server)
+
+#### Build and run
+
+This command will set up the environment and run the tests on demand.
+
+replace `/absolute/path/to/root/of/directory` after the -v flag below with the absolute path to the directory this project is stored in e.g. `/home/bob.smith/pyHS100`
+
+```shell
+docker build . -t pyhs100 && docker run -v /absolute/path/to/root/of/directory/pyHS100/tests:/opt/pyHS100/pyHS100/tests  pyhs100 pytest
```

Comment:
Considering that the build is being run inside the main directory, `/absolute/path/to/root/of/directory` part could be replaced with `$(PWD)/pyHS100/tests` to make it easier to use.

Reply:
yeah, I had that at first but then realised it won't work on Windows.

I could give one setup for Windows and one for Mac/Linux.

or I could just ignore Windows users :)

Let me know and I'll do that

Reply:
I would say that if someone is really developing something on Windows they pretty much know what they are doing, considering how difficult it all feels (to me at least). So it should be fine.

Reply:
I totally agree with @rytilahti 👍 I'm also in the same boat, I think python (and scripting languages in general) can be developed on Windows, but there's so many tools missing it requires a significant upfront investment to bring your environment en par with a UNIX one.

So yeah, if you're developing Python code on Windows, there's probably a good reason for it, and you should already have a good idea how to find your way around.